exporter/splunkhecexporter/config.go

```diff
@@ -46,6 +47,20 @@ type OtelToHecFields struct {
 	SeverityNumber string `mapstructure:"severity_number"`
 }
 
+// HecHeartbeat defines the heartbeat information for the exporter
+type HecHeartbeat struct {
+	// Interval represents the time interval for the heartbeat interval. If nothing or 0 is set,
+	// heartbeat is not enabled.
+	// A heartbeat is an event sent to _internal index with metadata for the current collector/host.
+	Interval time.Duration `mapstructure:"hec_heartbeat_interval"`
+
+	// ObservabilityName is the prefix will be used to observe the success or failure of the heartbeat
+	ObservabilityName string `mapstructure:"observability_name"`
```

Comment:
Is there no default value for that? Why is this need to be exposed as a user config? Can we have `otelcol_exporter_splunk_` which must be consistent across other components?

Reply:
That is true I need to add a default value here. Our application uses a specific metric name for other exporters for heartbeating so I want this customizable.

Reply:
`heartbeat_metric_name` would be clearer and defaulting to an internal exporter metric as suggested makes sense to me.

Reply:
The config option naming is also confusing because it's used as a prefix. And what if we want to add more metrics in the future not related to hec_heartbeat?

I think we should move these parameters to another section. Something like this:

```yaml
telemetry:
  enabled:
  metrics_prefix:
  extra_attributes:
```

I picked `telemetry` because is being used in collector core service for similar purposes.

WDYT? cc @rmfitzpatrick @atoulme

Reply:
yeah i think we can separate the telemetry section out. if no objection I will change the config to this^